core/src/main/java/bio/terra/pearl/core/service/publishing/PortalPublishingService.java

```diff
@@ -142,4 +150,39 @@ protected void applyChangesToNotificationConfigs(PortalEnvironment destEnv, List
         }
     }
 
+    protected void applyChangesToParticipantDashboardAlerts(PortalEnvironment destEnv, List<ParticipantDashboardAlertChange> changes) throws Exception {
+        for(ParticipantDashboardAlertChange change : changes) {
+            Optional<ParticipantDashboardAlert> destAlert = portalDashboardConfigService.findByPortalEnvIdAndTrigger(destEnv.getId(), change.trigger());
+            if(destAlert.isEmpty()) {
+                // The alert doesn't exist in the dest env yet, so default all the required fields before
+                // applying the changes from the change list
+                ParticipantDashboardAlert newAlert = ParticipantDashboardAlert.builder()
```

Comment:
Nit: It might be helpful to pull the default setup into it's own method, to emphasize this is the default, and to consolidate the code on lines 168-173.